Dockerfile.aggregation_job_creator

```diff
@@ -0,0 +1,14 @@
+FROM rust:1.60.0-alpine as builder
+RUN apk add libc-dev
+
+WORKDIR /src
+COPY Cargo.toml /src/Cargo.toml
+COPY Cargo.lock /src/Cargo.lock
+COPY janus_server /src/janus_server
+COPY monolithic_integration_test /src/monolithic_integration_test
+COPY test_util /src/test_util
+RUN --mount=type=cache,target=/usr/local/cargo/registry --mount=type=cache,target=/src/target cargo build --release --bin aggregation_job_creator && cp /src/target/release/aggregation_job_creator /aggregation_job_creator
+
+FROM alpine:3.15.4
```

Comment:
The `aggregator` Dockerfile copes in `/src/db/schema.sql`. Do we not need that here?

Reply:
It doesn't get used in the container at runtime, as we only copy it into the container for use by our Terraform external data provider. This is now moot as I'm going with a single Dockerfile.